.github/workflows/test.yml

```diff
@@ -2,32 +2,50 @@ name: Test
 
 on:
   push:
-    branches: ['*']
+    branches: [main]
   pull_request:
-    branches: ['*']
+    branches: [main]
```

Comment:
Does this mean it will only trigger when PRs are opened against main or the main branch is updated?

Reply:
Yep, and not only on opening new PRs, but also on new pushes to them.  At least that's how I'm used to use this most of the time to reduce load on CI, but if you intended some other cases that aren't matched hereby, I'm happy to revert this change ;)